runbotics/runbotics-orchestrator-ui/src/src-app/views/process/ProcessBuildView/Modeler/ActionFormPanel/widgets/DatePickerWidget/DatePickerWidget.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { FC, useState } from 'react';
+
+import { TextField } from '@mui/material';
+import { AdapterMoment } from '@mui/x-date-pickers/AdapterMoment';
+import { DateTimePicker } from '@mui/x-date-pickers/DateTimePicker';
+import { CalendarOrClockPickerView } from '@mui/x-date-pickers/internals/models';
+import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
+import { WidgetProps } from '@rjsf/core';
+import moment from 'moment';
+
+const inputTypes: {
+    [key: string]: { format: string; views: CalendarOrClockPickerView[] };
+} = {
+    date: {
+        format: 'YYYY-MM-DD',
+        views: ['year', 'month', 'day'],
+    },
+    'date-time': {
+        format: 'YYYY-MM-DD HH:mm',
+        views: ['year', 'month', 'day', 'hours', 'minutes'],
+    },
+    time: {
+        format: 'HH:mm',
+        views: ['hours', 'minutes'],
+    },
+    default: {
+        format: 'YYYY-MM-DD',
+        views: ['year', 'month', 'day'],
+    },
+};
```

Comment:
imo its better to add `as const` at the end of this object than type it this way